one_collect/src/helpers/exporting/os/linux.rs

```diff
@@ -692,11 +692,58 @@ impl OSExportMachine {
         Ok(())
     }
 
+    fn event_sampled_count_closure(
+        machine: &Writable<ExportMachine>,
+        session: &PerfSession,
+        callstack_reader: &CallstackReader,
+        kind: &str) -> impl FnMut(&EventData) -> anyhow::Result<()> + 'static {
+        let ancillary = session.ancillary_data();
+        let time_field = session.time_data_ref();
+        let pid_field = session.pid_field_ref();
+        let tid_field = session.tid_data_ref();
+        let reader = callstack_reader.clone();
+
+        /* Get sample kind for event */
+        let kind = machine.borrow_mut().sample_kind(kind);
+
+        /* Hook hard page fault event */
```

Comment:
I think this comment is a copy-paste bug since this is now in a separate function.